lib/src/directory_watcher/polling.dart

```diff
@@ -27,7 +27,9 @@ class PollingDirectoryWatcher extends ResubscribableWatcher
   PollingDirectoryWatcher(String directory, {Duration? pollingDelay})
       : super(directory, () {
           return _PollingDirectoryWatcher(
-              directory, pollingDelay ?? Duration(seconds: 1));
+            directory,
+            pollingDelay ?? const Duration(seconds: 1),
```

Comment:
Did we turn on a trailing comma lint? I wouldn't have expected that

Reply:
Sorry about that! That lint isn't enabled.

Since I was updating a few statements that ended up being split across lines, I tried adding them then. Definitely not necessary for this change though :)